Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/ml_impl_depth.cpp

```diff
@@ -1,20 +1,14 @@
 #include "ml_interface.hpp"
```

Comment:
This file can be removed. Was only used as an example of how the Interface can be implemented 😄